health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/appConfigurationRedesign/DependentFieldsWrapper.js

```
@@ -298,563 +335,627 @@ function DependentFieldsWrapper({
+                                                            }}
+                                                            style={{ cursor: "pointer" }}
+                                                        />
+                                                    ) : (
+                                                        <span
+                                                            style={{ cursor: "pointer", color: "#7F1D1D" }}
+                                                            onClick={() => {
+                                                                setGlobalFormError(null);
+                                                                setValidationStarted(false);
+                                                            }}
+                                                        >
+                                                            ×
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)